src/ImpactX.H

```diff
@@ -86,10 +86,20 @@ namespace impactx
          */
         void validate ();
 
+        /** Check inputs for unused parameters (e.g. typos) after step 1
+         */
+        void early_param_check ();
```

Comment:
Little idea to make the call-site more compact:
```suggestion
        /** Check inputs for unused parameters (e.g. typos) after step 1
         *
         * @return true if successful
         */
        void early_param_check ();
```